src/host_toolchains.py

```diff
@@ -26,8 +26,8 @@
 SETUP_TOOLCHAIN = os.path.join(CR_BUILD_DIR, 'toolchain', 'win',
                                'setup_toolchain.py')
 V8_SRC_DIR = os.path.join(WORK_DIR, 'v8', 'v8')
```

Comment:
I guess my question could have been more clear: Why did the original author every think it sue vs_toolchain.py from v8 rather than chromium (which would seem to be the more logically choice since chromium is effectively the upstream for these files).   Did the v8 version have some useful difference?  I think probably not but just curious really.